applications/views.py

```diff
@@ -174,3 +185,48 @@ def post(self, request, *args, **kwargs):
         application.save()
 
         return Response(status=status.HTTP_200_OK)
+
+
+class VideoInterviewsView(GenericAPIView):
+    """
+    Create an interview on Jobma, then redirect the user to it
+    """
+
+    authentication_classes = (SessionAuthentication,)
+    permission_classes = (IsAuthenticated, UserIsOwnerPermission)
+    lookup_field = "pk"
+    owner_field = "user"
+    queryset = BootcampApplication.objects.all()
+
+    def post(self, request, *args, **kwargs):
+        """
+        Create the Interview, then POST to Jobma to create it there,
+        then return a URL for the user to go to
+        """
+        user = request.user
+        application = self.get_object()
+        job = get_object_or_404(Job, run=application.bootcamp_run)
+        interview, _ = Interview.objects.get_or_create(
+            applicant=user, job=job, defaults={}
+        )
+        interview_link = create_interview(interview)
```

Comment:
1. Shouldn't we return immediately if the interview object already exists?
2. Is `defaults={}` necessary?
3. `create_interview` is an ambiguous name. Can you change that to `create_interview_in_jobma` or something like that?
4. Might be wrong about this, but it seems cleaner to me if we change `create_interview` to take a `Job` object and a `User` object (or simple kwargs for first/last/email instead of a `User). Those are the things it really needs to do its work, and I think it's better to avoid the side effect of setting the `interview_url` property and saving (it's clearer to the reader if we let the clients of this function do that)

Reply:
For the last point, the interview id is needed for the postback URL so we can't entirely get rid of it